NitroxServer/ConsoleCommands/ConfigCommand.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System.Diagnostics;
+using System.IO;
+using System.Reflection;
+using System.Threading;
+using System.Threading.Tasks;
+using NitroxModel.Core;
+using NitroxModel.DataStructures.GameLogic;
+using NitroxModel.Logger;
+using NitroxModel.Serialization;
+using NitroxServer.ConsoleCommands.Abstract;
+using NitroxServer.Serialization;
+
+namespace NitroxServer.ConsoleCommands
+{
+    internal class ConfigCommand : Command
+    {
+        private readonly SemaphoreSlim configOpenLock = new SemaphoreSlim(1);
+
+        public ConfigCommand() : base("config", Perms.CONSOLE, "Opens the server configuration file")
+        {
+        }
+
+        protected override void Execute(CallArgs args)
+        {
+            if (!configOpenLock.Wait(0))
+            {
+                Log.Warn("Waiting on previous config command to close the configuration file.");
+                return;
+            }
+
+            ServerConfig currentActiveConfig = NitroxServiceLocator.LocateService<ServerConfig>();
+            string configFile = Path.Combine(Path.GetDirectoryName(Assembly.GetEntryAssembly()?.Location) ?? "", currentActiveConfig.FileName);
+            if (!File.Exists(configFile))
+            {
+                Log.Error($"Could not find config file at: {configFile}");
+                return;
+            }
+
+            Task.Run(async () =>
+            {
+                try
+                {
+                    await StartProcessAsync("notepad", configFile);
```

Comment:
Is it possible to open the standard programm linked to the file type instead of notepad? Thinking about linux compatibility...

Reply:
Had to use pinvoke to get the standard program. As a fallback it iterates windows registry for the possible programs. If that can't be found it defaults to notepad. I added the code in a way that should make it easy to add cross platform support. Will throw `NotSupportedException` if left unimplemented so we'll notice it.